src/vm/methodtablebuilder.cpp

```diff
@@ -5090,10 +5111,15 @@ MethodTableBuilder::InitNewMethodDesc(
             NULL,
             NULL);
 
-        if (hr == S_OK)
+        if (hr == S_OK
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
```

Comment:
Could you please delete this ifdef here? It is not necessary and it will be become a problem for enabling the HW intrinsics on more platforms later.

Reply:
Done.